src/modules/lua_general.c

```diff
@@ -762,6 +784,14 @@ mtev_lua_general_init(mtev_dso_generic_t *self) {
   }
   lua_settop(lmc->lua_state, 0);
 
+  lua_getglobal(lmc->lua_state, "require");
+  lua_pushstring(lmc->lua_state, "ffi");
+  if(lua_pcall(lmc->lua_state,1,1,0) != 0 || !lua_istable(lmc->lua_state,-1)) {
+    mtevL(mtev_error, "lua_general could reference ffi\n");
+    return -1;
+  }
+  lmc->ffi_index = luaL_ref(lmc->lua_state, LUA_REGISTRYINDEX);
```

Comment:
`lmc->ffi_index` reference is never freed with `luaL_unref()`..

Reply:
This was intentional.. It lives until we close the lua state out for good.